aeron-client/src/main/java/io/aeron/CommonContext.java

```diff
@@ -419,6 +419,13 @@ public static InferableBoolean parse(final String value)
      */
     public static final String UNTETHERED_RESTING_TIMEOUT_PARAM_NAME = "untethered-resting-timeout";
 
+    /**
+     * Parameter name to set the max number of active retransmits for a publication
+     *
+     * @since 1.45.0
+     */
+    public static final String MAX_RETRANSMITS_PARAM_NAME = "retransmits-active-max";
```

Comment:
Naming. Why the value is `retransmits-active-max` and not simply `max-retransmits` or `retransmits-max`?
In any case the variable name should reflect the value like we have for other parameters.